src/glwidget.h

```diff
@@ -22,17 +22,16 @@
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
 
-
+#include <QOpenGLWidget>
```

Comment:
This, too, should be inside the `#include "pre_gaurd.h"` ... `#include "post_gaurd.h"` pair...